ui/components/src/ui-components/common/HighlightingComponentProps.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Common
+ */
+
+/**
+ * Record or Category match info used for identification of a specific match in record or category
```

Comment:
Need to fix documentation for these types. Should not mention records, properties or categories. Let's settle on "item" that covers [record, category, node, property] and "highlight" (instead of match)

Reply:
Changed it